libs/orchestrator-ui-components/src/lib/components/Subscription/FixedInputBlock.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiText } from '@elastic/eui';
+import { renderField } from './SubscriptionBlock';
+
+// Todo: add data type?
+export const FixedInputBlock = (title: string, data: object) => {
```

Comment:
`FixedInputBlock` component needs to be covered by both flavours of UI integration test(s) namely Jest snapshot tests and Storybook stories towards avoiding unintentional regressions both in the rendered HTML and the UI itself respectively.

Reply:
When #96 is done